modules/web/src/app/settings/admin/defaults/template.html

```diff
@@ -280,6 +294,17 @@
           <mat-checkbox [(ngModel)]="settings.webTerminalOptions.enableInternetAccess"
                         [disabled]="!settings.enableOIDCKubeconfig"
                         (change)="onSettingsChange()">
+            <mat-hint *ngIf="!settings.enableOIDCKubeconfig">This feature requires the OIDC kubeconfig feature to be enabled. Visit the
+              <a [href]="getDocumentationLink('enable-web-terminal-internet-access')"
```

Comment:
Notes for Reviewer:

TBD to Update screenshot and Info in Docs repo (Will add shortly)